pytest_splunk_addon/plugin.py

```diff
@@ -187,16 +194,36 @@ def init_pytest_splunk_addon_logger():
     """
     fh = logging.FileHandler(LOG_FILE)
     fh.setLevel(logging.INFO)
+    ch = logging.StreamHandler()
+    ch.setLevel(logging.WARNING)
     formatter = logging.Formatter(
         "%(asctime)s - %(levelname)s - %(filename)s - %(funcName)s - %(message)s"
     )
     fh.setFormatter(formatter)
+    ch.setFormatter(formatter)
     logger = logging.getLogger("pytest-splunk-addon")
     logger.addHandler(fh)
+    logger.addHandler(ch)
     logging.root.propagate = False
     logger.setLevel(logging.INFO)
     return logger
 
 
 init_pytest_splunk_addon_logger()
 LOGGER = logging.getLogger("pytest-splunk-addon")
+
+
+def pytest_exception_interact(node, call, report):
+    """
+    Hook called when an exception is raised during a test.
+    If the number of occurrences for a specific exception exceeds the limit in session.__exc_limits, pytest exits
+    https://docs.pytest.org/en/stable/reference/reference.html#pytest.hookspec.pytest_exception_interact
+    """
+    session = node.session
+    type_ = call.excinfo.type
+
+    if type_ in session.__exc_limits:
```

Comment:
add a comment:
It is ONLY an "Exception" as defined in EXC_MAP. EXC_MAP can be extended.